IDEAS/Buildings/Components/Window.mo

```diff
@@ -26,7 +26,7 @@ model Window "Multipane window"
   parameter Boolean linExtCon=sim.linExtCon
     "= true, if exterior convective heat transfer should be linearised (uses average wind speed)"
     annotation(Dialog(tab="Convection"));
-  parameter Boolean linExtRad=sim.linExtRad
+  parameter Boolean linExtRad=true
```

Comment:
I did not think about that.
It wouldn't make sense to change the default value, but then again neither would it for any of the other parameters so it does make sense to add it. I'll fix it.